application/classes/Ushahidi/Repository/User.php

```diff
@@ -133,6 +138,9 @@ public function isUniqueEmail($email)
 	// RegisterRepository
 	public function register(Entity $entity)
 	{
+
+		$this->updateIntercomUserCount(1);
```

Comment:
I think it'd make more sense to push the user info itself into Intercom when a new user is added, particularly admins. Then we can actually message all admins from a deployment via intercom.